src/embed_tests/TestRuntime.cs

```diff
@@ -91,6 +97,8 @@ public static void PyCheck_Iter_PyObject_IsIterable_ThreadingLock_Test()
         {
             Runtime.Runtime.Py_Initialize();
 
+            Runtime.Native.ABI.Initialize(Runtime.Runtime.PyVersion);
```

Comment:
What is this for?

Reply:
`PyIter_Check` down in the function is not something we import from Python DLL. In Python it is a macro, that we reimplemented in C#. That implementation depends on `TypeOffset` to be initialized, which is handled by `ABI.Initialize`